pkg/core/utils/utils.go

```diff
@@ -63,3 +67,23 @@ func strAt(slice []string, str string) int {
 	}
 	return -1
 }
+
+func SliceContains(in interface{}, elem interface{}) bool {
+	inValue := reflect.ValueOf(in)
```

Comment:
> Using reflect here looks overkill, the function name has already suggested that the input is a slice.

I didn't look clearly enough to realise the whole function is using reflection. I was trying to say using reflection for this function looks overkill. 

We don't actually have a case that needs to find a `interface{}` in a `[]interface{}`, so having a `StringSliceContains` is good enough for this case. This is where golang suffers when we want to use generic.